raiwidgets/raiwidgets/responsibleai_dashboard.py

```diff
@@ -48,41 +48,56 @@ def predict():
             return jsonify(self.input.on_predict(data))
         self.add_url_rule(predict, '/predict', methods=["POST"])
 
-        def tree():
-            data = request.get_json(force=True)
-            return jsonify(self.input.debug_ml(data))
-        self.add_url_rule(tree, '/tree', methods=["POST"])
-
-        def matrix():
-            data = request.get_json(force=True)
-            return jsonify(self.input.matrix(data))
-        self.add_url_rule(matrix, '/matrix', methods=["POST"])
-
-        def causal_whatif():
-            data = request.get_json(force=True)
-            return jsonify(self.input.causal_whatif(data))
-        self.add_url_rule(causal_whatif, '/causal_whatif', methods=["POST"])
-
-        def global_causal_effects():
-            data = request.get_json(force=True)
-            return jsonify(self.input.get_global_causal_effects(data))
-        self.add_url_rule(global_causal_effects, '/global_causal_effects',
-                          methods=["POST"])
-
-        def global_causal_policy():
-            data = request.get_json(force=True)
-            return jsonify(self.input.get_global_causal_policy(data))
-        self.add_url_rule(global_causal_policy, '/global_causal_policy',
-                          methods=["POST"])
-
-        def importances():
-            return jsonify(self.input.importances())
-        self.add_url_rule(importances, '/importances', methods=["POST"])
-
-        def get_exp():
-            data = request.get_json(force=True)
-            return jsonify(self.input.get_exp(data))
-        self.add_url_rule(get_exp, '/get_exp', methods=["POST"])
+        if analysis.task_type in [
```

Comment:
was this necessary for some reason, did it cause some issues? for text/vision/bigdata we don't do this type of thing AFAIK?

Reply:
No, I just thought there's no point in exposing defunct ways of accessing this Flask server.